### DISCUSSION TITLE
Playlist

### DISCUSSION THREAD

**Discussion by PaulHendriks** [funmusicplace/mirlo#819] [Ideas]
From a listeners point of view, creating playlists is a desirable feature.
Sharing lists with 'want to buy', or 'weekend mood', you name it.

It's not my intention to recreate the Spotify playlist hype, with editorial picks and such.

It would be nice for artists to show what they are listening to, who they hang out with.

**Comment by simonv3** [comment]
This is now do-able as described here https://mirlo.space/team/posts/Public-Playlists/

**Reply by PaulHendriks** [reply]
Thank you for adding this feature 😃
